### ISSUE TITLE
LayerNorm excluded from weight decay

### ISSUE THREAD

**Issue by SchenbergZY** [CyberZHG/keras-bert#60]
By seeing seperation of _kernel_weight_decay_ and _bias_weight_decay_ i can guarantee that bias has excluded from weight decay.
But are LayerNorm layers excluded? If yes that is not a feature request.

**Comment by CyberZHG** [comment]
Thanks. I've updated the optimizer.
